bedrock/base/templates/includes/structured-data/software/firefox-browser-software.json

```diff
@@ -0,0 +1,100 @@
+{
+  "@context": "https://schema.org/",
+  "@type": "SoftwareApplication",
+  "@id": "https://www.mozilla.org/#firefoxbrowser",
+  "url": "https://www.mozilla.org/firefox/new/",
+  "name": "Firefox Browser",
+  "image": "{{ settings.CANONICAL_URL + static('img/logos/firefox/quantum/logo-lg-high-res.png') }}",
+  "alternateName": [
+    "Firefox Desktop",
+    "Firefox Browser for Desktop",
+    "Mozilla Firefox"
+  ],
+  "sameAs": [
+    "https://www.wikidata.org/wiki/Q698",
+    "https://en.wikipedia.org/wiki/Firefox"
+  ],
+  "offers":
+  {
+    "@type": "Offer",
+    "price": "0",
+    "priceCurrency": "USD"
+  },
+  "OperatingSystem": [
+    "MacOS",
```

Comment:
Should this be `macOS`? (here and elsewhere)

Reply:
I think that doesnt matter but if we use it that way everywhere we should do it